ci/jenkins/jobs/macros.yaml

```diff
@@ -69,6 +69,40 @@
 
           if !(test -f jenkins/SECRET_EXIST); then
             echo "=== Fail to get secret ==="
+            
+            echo "=== Remove too many logs ==="
+            export LOG_DIR=/var/lib/jenkins/antrea_logs
+            dir_num=$(ls -lR ${LOG_DIR} | grep ^d | wc -l | awk '{print $1}')
+            if [[ ${dir_num} -ge 200 ]]; then
+              counter=0
+              ls -hltr ${LOG_DIR} | awk '{print $9}' | while read old_log; do
+                if [[ ${counter} -gt 0 ]]; then
+                  if [[ ${counter} -le 100 ]]; then
+                    rm -rf "${LOG_DIR}/${old_log}"
+                  else
+                    break
+                  fi
+                fi
+                let counter++
+              done
+            fi
+
+            echo "=== Saving capi logs ==="
+            mkdir -p ${LOG_DIR}/${cluster}/capi
+            kubectl get pod -n capi-system | awk '{print $1}' | while read capi_pod; do
```

Comment:
did you consider using `kubectl get -n capi-system pods -o name` here instead?

Reply:
Thanks. Updated.